elife/logrotate.sls

```diff
@@ -1,7 +1,14 @@
+logrotate noise filterer:
+    file.managed:
+        - name: /usr/local/bin/logrotate_noise_filter.py
+        - source: salt://elife/scripts/logrotate_noise_filter.py
```

Comment:
I see `scripts/` doesn't follow the convention that would name this `usr-local-bin-logrotate_noise_filter.py`, no problem.

Reply:
`scripts/` has always had much fewer files and their home isn't necessarily fixed to one location like a config file might be. It's a pedantic point and I don't mind renaming them if you find it easier